src/Tickets/QR/Settings.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace TEC\Tickets\QR;
+
+use WP_Error;
+
+/**
+ * Class Settings
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\QR
+ */
+class Settings {
+
+	/**
+	 * The option key that will be used to store the value.
+	 * This particular setting was originally stored in the Event Tickets Plus settings, now ETP should ways use the
+	 * option key here.
+	 *
+	 * @since TBD
+	 *
+	 * @var string The option key.
+	 */
+	protected static string $enabled_key = 'tickets-enable-qr-codes';
+
+	/**
+	 * The option key that will be used to store the value.
+	 * This particular setting was originally stored in the Event Tickets Plus settings, now ETP should ways use the
+	 * option key here.
+	 *
+	 * @since TBD
+	 *
+	 * @var string The option key.
+	 */
+	protected static string $api_hash_key = 'tickets-plus-qr-options-api-key';
+
+	/**
+	 * Get the option key that will be used to store the value.
+	 *
+	 * Option keys that are user controlled, need to be kept as public.
+	 *
+	 * @since TBD
+	 *
+	 * @return string
+	 */
+	public static function get_enabled_key(): string {
```

Comment:
I believe we should move all the settings to one place. Partially having some of them here, and then the rest duplicated in two plugins will make this piece really difficult to maintain. We could find a way to elegantly enable things from the "satellite" plugins.

Reply:
No, the goal is there is one QR code enabled setting. Not one per plugin.